File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/ClienteDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de los clientes para su devolucion
 * 
 * @author Flor
 *
 */
public class ClienteDTO {

	private String id;
	private String nif;
	private String nombre;
	private String direccion;
	private String poblacion;
	private String personaContacto;
	private String tlfContacto;
	private String email;
	private String idMetodoPago;
	private MetodoPagoDTO metodoPago;
	private String cuentaBancaria;
	private boolean baja;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getNif() {
		return nif;
	}

	public void setNif(String nif) {
		this.nif = nif;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getDireccion() {
		return direccion;
	}

	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}

	public String getPoblacion() {
		return poblacion;
	}

	public void setPoblacion(String poblacion) {
		this.poblacion = poblacion;
	}

	public String getPersonaContacto() {
		return personaContacto;
	}

	public void setPersonaContacto(String personaContacto) {
		this.personaContacto = personaContacto;
	}

	public String getTlfContacto() {
		return tlfContacto;
	}

	public void setTlfContacto(String tlfContacto) {
		this.tlfContacto = tlfContacto;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getIdMetodoPago() {
		return idMetodoPago;
	}

	public void setIdMetodoPago(String idMetodoPago) {
		this.idMetodoPago = idMetodoPago;
	}

	public MetodoPagoDTO getMetodoPago() {
		return metodoPago;
	}

	public void setMetodoPago(MetodoPagoDTO metodoPago) {
		this.metodoPago = metodoPago;
	}

	public String getCuentaBancaria() {
		return cuentaBancaria;
	}

	public void setCuentaBancaria(String cuentaBancaria) {
		this.cuentaBancaria = cuentaBancaria;
	}

	public boolean isBaja() {
		return baja;
	}

	public void setBaja(boolean baja) {
		this.baja = baja;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/ClienteDTO.cs

namespace Flotix2021.ModelDTO
{
	public class ClienteDTO
    {
        public ClienteDTO()
        {

        }

		public ClienteDTO(ClienteDTO clienteDTO)
		{
			_id = clienteDTO.id;
			_nif = clienteDTO.nif;
			_nombre = clienteDTO.nombre;
			_direccion = clienteDTO.direccion;
			_poblacion = clienteDTO.poblacion;
			_personaContacto = clienteDTO.personaContacto;
			_tlfContacto = clienteDTO.tlfContacto;
			_email = clienteDTO.email;
			_idMetodoPago = clienteDTO.idMetodoPago;
			_metodoPago = clienteDTO.metodoPago;
			_cuentaBancaria = clienteDTO.cuentaBancaria;
			_baja = clienteDTO.baja;
		}

		private string _id;

		public string id
		{
			get { return _id; }
			set { _id = value; }
		}

		private string _nif;

		public string nif
		{
			get { return _nif; }
			set { _nif = value; }
		}

		private string _nombre;

		public string nombre
		{
			get { return _nombre; }
			set { _nombre = value; }
		}

		private string _direccion;

		public string direccion
		{
			get { return _direccion; }
			set { _direccion = value; }
		}

		private string _poblacion;

		public string poblacion
		{
			get { return _poblacion; }
			set { _poblacion = value; }
		}
		
		private string _personaContacto;

		public string personaContacto
		{
			get { return _personaContacto; }
			set { _personaContacto = value; }
		}
		
		private string _tlfContacto;
		
		public string tlfContacto
		{
			get { return _tlfContacto; }
			set { _tlfContacto = value; }
		}
		
		private string _email;

		public string email
		{
			get { return _email; }
			set { _email = value; }
		}
		
		private string _idMetodoPago;

		public string idMetodoPago
		{
			get { return _idMetodoPago; }
			set { _idMetodoPago = value; }
		}
		
		private MetodoPagoDTO _metodoPago;

		public MetodoPagoDTO metodoPago
		{
			get { return _metodoPago; }
			set { _metodoPago = value; }
		}
		
		private string _cuentaBancaria;

		public string cuentaBancaria
		{
			get { return _cuentaBancaria; }
			set { _cuentaBancaria = value; }
		}
		
		private bool _baja;

		public bool baja
		{
			get { return _baja; }
			set { _baja = value; }
		}
	}
}
